src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.skia.cs

```diff
@@ -18,6 +18,9 @@ namespace Windows.UI.Xaml.Shapes
 {
 	partial class BorderLayerRenderer
 	{
+		private static SKPoint[] _outerRadiiStore = new SKPoint[4];
+		private static SKPoint[] _innerRadiiStore = new SKPoint[4];
```

Comment:
candidate for `stackalloc` inside the methods ?
It would avoid potential multithreading issue (even if UI code should not use multiple threads)

Reply:
I think `stackalloc` does not allow assigning to an array, or does it? The reason is that I need an array to be able to pass to the Skia `SetRectRadii` method

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-7.3/stackalloc-array-initializers

Reply:
Will keep as is for now and will adjust in future PR which will also include WASM

Reply:
@spouliot I wasn't able to figure out the syntax - based on the linked article, the target can be `Span<T>` or `T*`, but not sure how to get an actual array

Reply:
>  Skia SetRectRadii method

sorry missed that part :( and thought you controlled the usage of the variable